include/aktualizr-lite/api.h

```diff
@@ -221,6 +221,8 @@ class AkliteClient {
    */
   CheckInResult CheckIn() const;
 
+  CheckInResult CheckInLocal(const std::string path) const;
```

Comment:
Please add a unit test for this new check in method similar to the unit test for the remote checkin https://github.com/foundriesio/aktualizr-lite/blob/e5106f240b4e2c27414e36c855d5d54ab33ab7e0/tests/apiclient_test.cc#L102.

Reply:
Done. Strongly based on the `CheckIn` test.